packages/openapi-2-kong/src/index.js

```diff
@@ -30,7 +31,7 @@ export async function generateFromString(
   tags: Array<string> = [],
 ): Promise<ConversionResult> {
   const api: OpenApi3Spec = await parseSpec(specStr);
-  return generateFromSpec(api, type, ['OAS3_import', ...tags]);
+  return generateFromSpec(api, type, [defaultTags.join(), ...tags]);
```

Comment:
Is the `.join()` necessary? Doing the following should be enough

```suggestion
  return generateFromSpec(api, type, [...defaultTags, ...tags]);
```

Or even
```suggestion
  return generateFromSpec(api, type, defaultTags.concat(tags));
```